### DISCUSSION TITLE
resolution reduction

### DISCUSSION THREAD

**Discussion by N3rvi** [skyline-emu/skyline#1340] [Ideas]
please make it so that it would be possible to reduce the resolution for example to 540p

**Comment by JERD001** [comment]
That's what dock/handheld modes are for, cheats for even lower resolution are kinda unnecessary. That just mean your phone is not capable of actually using the emulator/demanding game in the first place.

**Reply by kingkartick** [reply]
So what Everyone should have a chance that's what mobile gaming is

**Reply by JERD001** [reply]
Mobile gaming isn't about everyone should have a chance. If your phone isn't capable of handling a 3D game in handheld mode were 90% of the titles are sub 720p that just means the phone isn't capable of pulling playing the game.

**Reply by kingkartick** [reply]
Still skyline needs to have some settings for lower resolution because everyone does not. Have a flagship chip and minimum requirements is snapdragon 845 which is still high for 2021 's midrange phones , by midrange I mean 200 bucks , otherwise I will buy a switch rather than this

**Reply by rebornsunny2000** [reply]
uhhhhh you actually NEED a switch to use this emulator though???

**Reply by JERD001** [reply]
> Still skyline needs to have some settings for lower resolution because everyone does not. Have a flagship chip and minimum requirements is snapdragon 845 which is still high for 2021 's midrange phones , by midrange I mean 200 bucks , otherwise I will buy a switch rather than this

You need a switch in the first place to legally use skyline. Snapdragon 845 is the minimum requirements for a reason.
